### ISSUE TITLE
Stop the mapchanging.

### ISSUE THREAD

**Issue by darkgusgus** [darkgusgus/HungerGames#16]
If there is no one playing. It would be nice to have the server stop rotating maps. changing maps every minute if you're just talking is quite annoying.

ideas:
/callvote pause

**Comment by darkgusgus** [comment]
Been playing around with maprotation.cfg. I think the answer to this is there, trying to throw in restarts with a numclients ¨< 3 . or something like that. Haven't been succesful yet.

**Comment by Foe-of-Eternity** [comment]
Perhaps there could be a warmup until enough players join, and then restarts into the live game?

**Comment by darkgusgus** [comment]
That is an excellent idea. Trigger the SD countdown only once the warmup is done.  (or something like that) Needs elaboration.
